.github/workflows/test-assistant-pr-review.yml

```diff
@@ -0,0 +1,44 @@
+name: Send PRs that require review to Slack
+
+on:
+  pull_request:
```

Comment:
This should probably be `pull_request_target` otherwise this event won't work for community PRs.

Reply:
Great point! Thanks @ObliviousHarmony